### ISSUE TITLE
Already downloaded maps not recognized in some cases

### ISSUE THREAD

**Issue by uspoma** [mapsme/omim#8964]
Hello.

The problem seems as this: Once I reinstall maps.me it does not find already installed maps and wants to re-download all again.

To reproduce the problem:
1. Install maps.me
2. Let it download the initial world map
3. Go to settings and set the data folder to an external sd-card
4. If it asks to move data to the new path choose yes
5. Download a random map
6. Remove Maps.me
7. Re-Install maps.me
=> At this point maps.me seems not able to use already downloaded maps. 

A) it wants to re-download the world-map. There should be a way to use the one already downloaded.
B) Once the world map is downloaded again and I go to settings and choose the external sd-card it does not recognize the maps that are stored there. So I have to re-download all the files again which cost time and bandwidth and attention to the process...

Could you sort these problems out? Thanks by the way for working on such a great app!

Maps.Me: 8.2.3
Android: 7.1.2

**Comment by Alexuser1** [comment]
Hello @uspoma ,

When you download the app from google play the world map is downloaded automatically. There is no need to download it separately.

If you move downloaded maps to SD via the app settings - save maps to, all downloaded maps will be deleted when you uninstall the app.

Please specify where do you store the app itself: in the internal storage or SD card?

Thanks,
Alexander

**Comment by uspoma** [comment]
Hello @Alexuser1 

Thanks for getting back swiftly.

I installed Maps.Me on the internal storage. The maps are on the SD-card.
Now that you mentioned it - Maps.Me seems to have removed all maps when I removed and re-installed it.

I guess this is a design decision. Weather or not this is a good decision (maybe downloaded maps on external storage should survive) might be a different ticket but I am going to close this one.

Thanks.
